app/code/Magento/Sales/Test/Unit/Model/Order/ShipmentTest.php

```diff
@@ -6,6 +6,7 @@
 namespace Magento\Sales\Test\Unit\Model\Order;
 
```

Comment:
For this fix, the unit test is not enough. The fix should be covered at least by integration test. The unit test doesn't check even simple flow when a comment can be added to the shipment. For more details, please, see my https://github.com/magento/magento2/pull/16248#discussion_r199861204.